### PR TITLE
Improve RST and MD files

### PR DESCRIPTION
**Pull request by felixfontein**
Subset of #1287 that doesn't add linting.

#1272 should be merged first and this PR then updated.

### CODE REVIEW

**Comment by hiddeco** [comment]
@felixfontein any chance you would be up to addressing the mixed usage of Sops / SOPS / sops throughout the document in this PR? In my opinion, it should be either "SOPS" (the name) or `sops` (the binary) depending on the context.

**Comment by felixfontein** [comment]
@hiddeco good idea, I added a commit which normalize the SOPS usages. I converted quite a few `sops` to SOPS as well. Please take a look whether you think that's OK.

File: README.rst [hunk 1]
```diff
@@ -331,79 +333,79 @@ We assume you have an instance (or more) of Vault running and you have privilege
...
+          hc_vault_transit_uri: "$VAULT_ADDR/v1/sops/keys/thirdkey"
+    EOF
+
+    $ sops --verbose -e prod/raw.yaml > prod/encrypted.yaml
 
 Adding and removing keys
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-When creating new files, ``sops`` uses the PGP, KMS and GCP KMS defined in the
+When creating new files, SOPS uses the PGP, KMS and GCP KMS defined in the
```
Comment (hiddeco) [inline README.rst:392]: As we later make mention of flags, maintaining the command reference is better here I think.

There are a couple of other places where this applies as well.

Comment (felixfontein) [reply README.rst]: I hope I manged to fix the right ones you meant.

**Review by hiddeco — APPROVED** [review]

**Comment by felixfontein** [comment]
@hiddeco thanks for reviewing and merging!

**Pull request merged**